src/components/ButtonWithDropdownMenu/index.tsx

```diff
@@ -245,6 +245,7 @@ function ButtonWithDropdownMenu<IValueType>({
                     headerText={menuHeaderText}
                     menuItems={options.map((item, index) => ({
                         ...item,
+                        // Clear shouldShowRightIcon since PopoverMenu doesn't use this property
```

Comment:
`PopoverMenu` doesn't, but props here get passed to `FocusableMenuItem`, right? I don't quite understand this comment, sorry

Reply:
yeah that could be cleanest / easiest to understand 👍